Demo/UI Tests/Venmo UI Tests/Venmo_UITests.swift

```diff
@@ -4,60 +4,16 @@ class Venmo_UITests: XCTestCase {
 
     // swiftlint:disable implicitly_unwrapped_optional
     var demoApp: XCUIApplication!
-    var mockVenmo: XCUIApplication!
     // swiftlint:enable implicitly_unwrapped_optional
 
     override func setUp() {
         super.setUp()
         continueAfterFailure = false
 
-        mockVenmo = XCUIApplication(bundleIdentifier: "com.braintreepayments.MockVenmo")
-        mockVenmo.activate()
-
         demoApp = XCUIApplication(bundleIdentifier: "com.braintreepayments.Demo")
         demoApp.launchArguments.append("-EnvironmentSandbox")
         demoApp.launchArguments.append("-ClientToken")
         demoApp.launchArguments.append("-Integration:VenmoViewController")
         demoApp.launch()
     }
-    
-    func testTokenizeVenmo_whenSignInSuccessfulWithPaymentContext_returnsNonce() {
```

Comment:
We should add a TODO + JIRA ticket for investigating what we want to do about UI tests now that our MockVenmo app will no longer work. (Also likely removing the MockVenmo app entirely, unless we can rig it to mimic Venmo's universal link)

Reply:
Good point, done!

Reply:
Ticket https://paypal.atlassian.net/jira/software/c/projects/DTMOBILES/boards/27751?selectedIssue=DTMOBILES-1237